bee-common/bee-common/tests/lib.rs

```diff
@@ -1,4 +1,4 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-fn main() {}
+mod packable;
```

Comment:
Is this file needed ?

Reply:
I don't follow, everything at the root of `tests` is executed with or without lib.rs

Reply:
oh wow, then yes we don't need it :P